packages/create-cloudflare/src/helpers/args.ts

```diff
@@ -54,7 +53,12 @@ export const parseArgs = async (argv: string[]): Promise<Partial<C3Args>> => {
 		})
 		.option("existing-script", {
 			type: "string",
-			hidden: templateMap["pre-existing"].hidden,
+			hidden: true,
+		})
+		.option("template", {
+			type: "string",
+			description:
+				"The url of a git repository containing the C3 template to run",
```

Comment:
is `template to run` the correct way of putting this?

(or is this a leftover from the initial more generic implementation?)